packages/api/src/schema/api-schema.yaml

```diff
@@ -768,8 +805,10 @@ components:
           example: de7818e7-610a-4057-8f6f-b785dc1e6f88
           description: Points to parent stream object
         record:
-          description: |
-            Whether the stream should be recorded. Uses default settings. For more
+          description: >
+            Whether the stream should be recorded. Uses default settings. For
+            more
+
```

Comment:
I don't know if this is an issue, but just flagging that there's an extra newline here